shell-ui/src/NavBar.js

```diff
@@ -62,8 +69,68 @@ const translateOptionsToMenu = (
   );
 };
 
-export const Navbar = ({ options, userGroupsMapping }: { options: Options, userGroupsMapping?: UserGroupsMapping }): Node => {
+const NavbarDropDownItem = styled.div`
+  display: flex;
+  width: 200px;
+  align-items: center;
+`;
+
+const NavbarDropDownItemIcon = styled.div`
+  padding: 10px;
```

Comment:
I think it makes sense to remove this padding since we already defined 16px padding for the item in core-ui.
It looks too big for me.

Same for `<NavbarDropDownItemExternal/>`
![image](https://user-images.githubusercontent.com/18453133/112858740-71a2b880-90b2-11eb-9f05-bc0f0a51026b.png)